docs/roadmap/planning.rst

```diff
@@ -0,0 +1,37 @@
+:title: Planning Process
+:description: The open source planning process for the Deis project used to define the roadmap and its execution.
+
+.. _planning:
+
+Planning Process
+================
+Deis features a lightweight process that emphasizes openness and ensures every community member can be an integral part of the planning process.
```

Comment:
Maybe end with "planning for the future" instead? Then we don't repeat the word "process" in this sentence.